python/pyspark/sql/connect/column.py

```diff
@@ -508,45 +508,24 @@ def contains(self, other: Union[PrimitiveType, "Column"]) -> "Column":
         """
         return _bin_op("contains")(self, other)
 
-    def startswith(self, other: Union[PrimitiveType, "Column"]) -> "Column":
-        """
-        String starts with. Returns a boolean :class:`Column` based on a string match.
-
-        Parameters
-        ----------
-        other : :class:`Column` or str
-            string at start of line (do not use a regex `^`)
-
-        Examples
-        --------
-        >>> df = spark.createDataFrame(
-        ...      [(2, "Alice"), (5, "Bob")], ["age", "name"])
```

Comment:
This does not work in current Connect `createDataFrame` which does not allow a `schema`

Reply:
what about just removing the examples, if we can not support them now.

Reply:
A bit messy on our examples. Ideally every examples should be tested and proven to work....